osbs/build/build_request.py

```diff
@@ -96,6 +99,7 @@ def set_params(self, **kwargs):
 
         self.base_image = kwargs.get('base_image')
         self.low_priority_node_selector = kwargs.get('low_priority_node_selector')
+        self.platform_node_selector = kwargs.get('platform_node_selector')
```

Comment:
if you do 

self.platform_node_selector = kwargs.get('platform_node_selector', {})

you don't need the if,else on 1101